pkg/service/init.go

```diff
@@ -322,7 +322,9 @@ func RunE2E(clientset *kubernetes.Clientset) {
 
 // Cleanup removes all resources created during E2E tests.
 func Cleanup(clientset *kubernetes.Clientset) {
+	common.SetDefaultNamespace()
```

Comment:
@Bharadwajshivam28 I think we can call this from cmd/root.go before calling `Cleanup`

Reply:
can you please make me understand this? I think you are saying to call this `common.SetDefaultNamespace()` from the root.go?

Reply:
correct, here https://github.com/kubernetes-sigs/hydrophone/blob/main/cmd/root.go#L61

Reply:
oh.. you are saying to remove the line `common.SetDefaultNamespace()` from `init.go` and call it from here i```
f cleanup {
			common.SetDefaultNamespace()
			service.Cleanup(client.ClientSet)
```

from `root.go` file

Reply:
yes